test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -414,7 +418,7 @@ output`;
 
 				// Redirect a print statement to the Output widget.
 				await app.workbench.positronConsole.pasteCodeToConsole(`with output:
-    print('Hello, world!')
+print('Hello, world!')
```

Comment:
This is the change that breaks for me.   Without the indentation, I get
```
>>> with output:
... print('Hello, world!')
IndentationError: expected an indented block after 'with' statement on line 1 (4154732719.py, line 2)
```
Just curious why it was removed?

Reply:
Oh weird. I got an error on the husky pre-commit hook about the white space here. 
> test/smoke/src/areas/positron/plots/plots.test.ts(421,1): Bad whitespace indentation

Which is why I edited it and then it was happy. Soooo? I can put it back but I'll just have to pass in `--no-verify` to skip the checks... First tho, I'll try reverting that change in the morning and confirm I wasn't imagining the precommit failure.